tslint.json

```diff
@@ -7,6 +7,7 @@
     "no-implicit-dependencies": [true, "dev"],
     "no-shadowed-variable": false,
     "no-submodule-imports": [true],
-    "no-unused-variable": true
+    "no-unused-variable": true,
+    "semicolon": [true, "always", "ignore-interfaces"]
```

Comment:
Is this necessary?

Reply:
Not necessary but useful I would say.

Reply:
Let's not change it, since prettier inserts semicolons there anyway.